app.py

```diff
@@ -226,7 +226,7 @@ def generate_graph(street, direction, day_type='Weekday', period='AMPK'):
     '''
     after_data, base_data = filter_graph_data(street, direction, day_type, period)
     if after_data.empty:
-        data = [go.Bar()]
+        data = html.Div(className = 'nodata')
```

Comment:
While this makes the graph not appear, a large white space remains because this empty `dcc.Graph` is created in `app.layout()` at https://github.com/CityofToronto/bdit_king_pilot_dashboard/blob/3fe25b0693ee0886a9bf210b04b2182fd2c182d3/app.py#L338-L340